src/sentry/integrations/msteams/webhook.py

```diff
@@ -264,18 +264,11 @@ def _get_team_installation_request_data(cls, data: dict[str, Any]) -> dict:
         We want the KeyError exception to be raised if the key does not exist.
         """
         channel_data = data["channelData"]
-        new_team_info = channel_data["team"]
-
-        team_id = new_team_info.get("aadGroupId", None)
-        if team_id is None:
-            logger.info(
-                "sentry.integrations.msteams.webhooks: New team info data does not have aadGroupId",
-                extra={"data": data},
-            )
-            fallback_id = new_team_info["id"]
-            team_id = fallback_id
 
+        new_team_info = channel_data["team"]
+        team_id = new_team_info["id"]
```

Comment:
this will always be there?

Reply:
it should based on the docs!

Reply:
if not, we will raise and catch the exception, responding back with a 400